tests/test_treeherder_menu.py

```diff
@@ -13,12 +13,12 @@ def test_treeherder_dropdown(base_url, selenium):
     treeherder_page = TreeherderPage(selenium, base_url).open()
 
     # Verify Treeherder loads
-    assert treeherder_page.unclassified_failure_count > 0
+    assert treeherder_page.result_sets > 0
 
     # Switch to Perfherder page
     perfherder_page = treeherder_page.open_perfherder_page()
     assert perfherder_page.is_graph_chooser_displayed
 
     # Return to Treeherder page
     treeherder_page = perfherder_page.open_treeherder_page()
-    assert treeherder_page.unclassified_failure_count > 0
+    assert treeherder_page.result_sets > 0
```

Comment:
As above, this is just duplicating the wait that's already called so I would say it's safe to just remove this line.